egs/librispeech/ASR/zipformer/zipformer.py

```diff
@@ -1499,7 +1567,7 @@ def forward(
 
         # self-attention
         q = x[...,0:query_dim]
-        k = x[...,query_dim:2*query_dim]
+        k = y[...,query_dim:2*query_dim]
```

Comment:
The slice index would need to be changed as in above comment

Reply:
OK